src/DotLiquid.Tests/StandardFilterTests.cs

```diff
@@ -1487,10 +1487,19 @@ public void TestUrldecode()
         [Test]
         public void TestDefault()
         {
-            Hash assigns = Hash.FromAnonymousObject(new { var1 = "foo", var2 = "bar" });
+            Hash assigns = Hash.FromAnonymousObject(new { var1 = "foo", var2 = "bar", array = new []{"1", "2"}, empty_array = new string[]{} });
             Helper.AssertTemplateResult("foo", "{{ var1 | default: 'foobar' }}", assigns);
             Helper.AssertTemplateResult("bar", "{{ var2 | default: 'foobar' }}", assigns);
             Helper.AssertTemplateResult("foobar", "{{ unknownvariable | default: 'foobar' }}", assigns);
+            Helper.AssertTemplateResult("foobar", "{{ false | default: 'foobar' }}", assigns);
+            Helper.AssertTemplateResult("foobar", "{{ '' | default: 'foobar' }}", assigns);
+            Helper.AssertTemplateResult("foobar", "{{ empty_array | default: 'foobar' }}", assigns);
+            Helper.AssertTemplateResult("12", "{{ array | default: 'foobar' }}", assigns);
+            Helper.AssertTemplateResult("1", "{{ 1 | default: 100 }}", assigns);
+
+            // allow false parameter
+            Helper.AssertTemplateResult("foobar", "{{ false | default: 'foobar', false }}", assigns);

```

Comment:
documentation shows that the correct syntax is with explicit named parameter, e.g. `, allow_false: true`

Reply:
See https://replit.com/@microalps/LiquidUnitTests#main.rb (test_default_ignores_two_args) for proof that a second argument is ignored and not treated the same as a named argument. So, this doesn't just not match the documentation, it is actually a false assertion.